content/en/faqitems/fleet-management-1-how-do-i-ssh-into-bottlerocket.markdown

```diff
@@ -0,0 +1,9 @@
++++
+question = "How do I SSH into Bottlerocket?"
+group = "Fleet Management"
```

Comment:
Change the group to "Access" (this will also affect the filename)

I'd like to avoid the proliferation of many subtly different categories.